_posts/2021-06-06-announcing_trino_summit.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Announcing Trino Summit"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Greetings Trino Nation,
+
+As mentioned in [last year’s Presto Summit blog](https://blog.starburst.io/presto-summit-sf-postponed-to-2021), 
+“It’s not goodbye, it’s see you later!” We may not have guessed at the time that
+“later” was a year and a half, but here we are! Get ready for this year’s hybrid
+Trino Summit event! This year’s Summit feels a little different coming back for 
+many reasons. For one, the name of the event has changed from Presto to Trino, 
+which means this will be the first event of the project hosted 
+[under the new banner of Trino](https://trino.io/blog/2020/12/27/announcing-trino.html)! 
+
+<!--more-->
+
+New names and cute bunnies aside, this event has a lot of extra excitement as we
+ring in the first hybrid in-person + virtual Trino event held since the 
+beginning of the pandemic! We are really looking forward to talking in front of
+a live audience and the high fives, hugs, and hand shakes we’ve all missed for 
+over a year.
+
+This year's Summit is hosted at [the Commonwealth Club](https://www.commonwealthclub.org/visitorinfo)
+in San Francisco, CA on October 13th. We had originally set the date on 
+September 15th but later realized that this was conflicting with Yom Kippur. 
+While making these updates, it seemed clear to move this event to be a hybrid 
+format so we will be reaching out to those who have already registered. If you 
```

Comment:
"it became clear that we could include more people with a hybrid format"

When we say "we will reach out to those who have already registered." - what will we reach out with? Maybe we can end the sentence after "hybrid format." and continue on with "If you've already registered, we'll reach out to you with updated information and attendance options."